vta/src/sim/sim_driver.cc

```diff
@@ -196,7 +196,7 @@ class DRAM {
 
  private:
   // The bits in page table
-  static constexpr vta_phy_addr_t kPageBits = 12;
+  static constexpr vta_phy_addr_t kPageBits = VTA_PAGE_BITS;
   // page size, also the maximum allocable size 16 K
   static constexpr vta_phy_addr_t kPageSize = 1 << kPageBits;
```

Comment:
replace with VTA_PAGE_BYTES macro for completeness

Reply:
fixed.